commitchecker/README.md

```diff
@@ -16,3 +16,44 @@ from https://github.com/openshift/kubernetes/tree/0abcd84431df81ed9f2a1846b5045e
 
 That repository can't be vendored, and you can't use `go get` against it (for multiple reasons), so we extracted the
 commitchecker code and moved it here.
+
+# Usage
+
+## On command line
+
+```
+commitchecker:
+  -end string
+	The end of the revision range for analysis (default "HEAD")
+  -start string
+	The start of the revision range for analysis (default "main")
+```
+
+## In OpenShift CI
+
+In your repository configuration (`github.com/openshift/release/ci-operator/config/$org/$repo/*.yaml`):
+
+1. Import commitchecker image from `ci` namespace:
+
+```yaml
+base_images:
+...
+  commitchecker:
+    name: commitchecker
+    namespace: ci
+    tag: "4.14"  # use appropriate OCP branch
+```
+
+2. Add `verify-commits` presubmit CI job that clones your repository, applies a PR, runs `commitchecker` image in a container and runs `commitchecker` inside:
+
+```yaml
+tests:
+...
+
+- as: verify-commits
+  commands: |
+    commitchecker --start ${PULL_BASE_SHA:-master}
+  container:
+    clone: true
```

Comment:
According to the docs for this field, it defaults to true if the `from` image is in `base_images` FYI

Reply:
Indeed, I removed `clone: true` as unnecessary.

@ncdc PTAL, I copied the example from [the AWS EBS CSI driver jobs](https://github.com/openshift/release/pull/42462/files)

Reply:
👍 yeah, we've turned these on for our openshift/operatator-framework-{catalogd, operator-controller, rukpak} repos. Thanks again for turning this into a reusable image!